app/controllers/tasks_controller.rb

```diff
@@ -1,8 +1,52 @@
 # frozen_string_literal: true
 
 class TasksController < ApplicationController
-  layout false
-  def index; end
+  before_action :require_login
 
-  def overview; end
+  before_action :find_task, only: %i[show edit update destroy]
+
+  def index
+    @tasks = Task.where(task_list_id: params[:id]).where(done: false)
+  end
+
+  def show; end
```

Comment:
Is this used? I don't see any view for it. If it's not used then we should remove it and ideally also skip it while generating routes, see https://guides.rubyonrails.org/routing.html#restricting-the-routes-created